community/15-shenyu-client-dotnet-release-guide.md

```diff
@@ -291,36 +291,34 @@ dev@shenyu.apache.org
 Title:
 
 ```
-[VOTE] Release Apache ShenYu .NET ${PUBLISH.VERSION}
+[VOTE] Release Apache ShenYu .NET client ${PUBLISH.VERSION}
 ```
 
 Content:
 
 ```
 Hello ShenYu Community,
 
-This is a call for vote to release Apache ShenYu .NET version ${PUBLISH.VERSION}
+This is a call for vote to release Apache ShenYu Client .NET version 1.0.0.
```

Comment:
not 1.0.0.     ${PUBLISH.VERSION} is right

Reply:
resolved.